subprojects/logging/src/test/groovy/org/gradle/internal/logging/sink/ProgressLogEventGeneratorTest.groovy

```diff
@@ -17,468 +17,145 @@
 
 package org.gradle.internal.logging.sink
 
+import org.gradle.api.logging.LogLevel
+import org.gradle.internal.SystemProperties
 import org.gradle.internal.logging.OutputSpecification
+import org.gradle.internal.logging.events.EndOutputEvent
+import org.gradle.internal.logging.events.OperationIdentifier
 import org.gradle.internal.logging.events.OutputEventListener
-import org.gradle.internal.logging.events.StyledTextOutputEvent
+import org.gradle.internal.logging.events.ProgressCompleteEvent
+import org.gradle.internal.logging.events.ProgressStartEvent
+import org.gradle.internal.progress.BuildOperationType
+import org.gradle.util.MockTimeProvider
 
 class ProgressLogEventGeneratorTest extends OutputSpecification {
```

Comment:
Ok, I have exposed package-private getters to the map and asserted their state at the end of the build at least to prevent memory leaks.